class/defaults.yml

```diff
@@ -76,14 +70,22 @@ parameters:
           # Only configure scheduler PDB if the scheduler is actually enabled.
           # Otherwise we get unnecessary alert noise for the PDB because no
           # scheduler pods are running.
-          enabled: ${topolvm:helmValues:scheduler:enabled}
+          enabled: ${topolvm:helm_values:scheduler:enabled}
 
       # lvmd service
       lvmd:
         # lvmd.managed -- If true, set up lvmd service with DaemonSet.
         managed: true
         # lvmd.resources -- Specify resources.
         resources: ${topolvm:resources:lvmd}
+        # lvmd.deviceClasses -- Specify the device-class settings.
+        deviceClasses: ${topolvm:deviceclasses}
+        # lvmd.nodeSelector -- Specify nodeSelector.
+        nodeSelector:
+          syn.tools/topolvm: ""
+
+      # storageClasses -- Whether to create storageclass(s)
+      storageClasses: []
```

Comment:
There is no reference to ${topolvm:storageclasses}. Is this intended? Why is it an hash first and a list later?

Reply:
Yes I create the storage class using the storageclass component.
https://github.com/projectsyn/component-topolvm/blob/master/component/main.jsonnet#L131